hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -101,7 +101,8 @@ enum Type {
 
   RevokeS3Secret = 93;
 
-  PurgeDirectories = 94;
+  PurgePaths = 94; // [deprecated=true]
```

Comment:
Any specific reason for commented out the statement. Should it be 

`PurgePaths = 94  [deprecated=true];`

Reply:
deprecated keyword is not supported for enum in protobuf 2.5